webapp/components/sidebar.jsx

```diff
@@ -370,6 +370,13 @@ export default class Sidebar extends React.Component {
         this.setState({showDirectChannelsModal: false});
     }
 
+    openLeftSidebar() {
+        if (Utils.isMobile()) {
+            $('.app__body .inner-wrap').toggleClass('move--right');
```

Comment:
Not sure, we're already doing it with jquery in other parts of the app. If someone knows how to do it just let me know and I'll change it.